ydb/public/sdk/cpp/include/ydb-cpp-sdk/client/rate_limiter/rate_limiter.h

```diff
@@ -2,14 +2,53 @@
 
 #include <ydb-cpp-sdk/client/driver/driver.h>
 
+#include <library/cpp/json/json_value.h>
+
+#include <unordered_map>
+#include <variant>
+
 namespace Ydb::RateLimiter {
-class CreateResourceRequest;
-class DescribeResourceResult;
-class HierarchicalDrrSettings;
+    class CreateResourceRequest;
+    class DescribeResourceResult;
+    class HierarchicalDrrSettings;
+    class ReplicatedBucketSettings;
+    class MeteringConfig;
+    class MeteringConfig_Metric;
 } // namespace Ydb::RateLimiter
 
 namespace NYdb::inline V3::NRateLimiter {
 
+struct TReplicatedBucketSettings {
+    using TSelf = TReplicatedBucketSettings;
+
+    TReplicatedBucketSettings() = default;
+    TReplicatedBucketSettings(const Ydb::RateLimiter::ReplicatedBucketSettings&);
+
+    void SerializeTo(Ydb::RateLimiter::ReplicatedBucketSettings&) const;
+
+    // Interval between syncs from kesus and between consumption reports.
+    // Default value equals 5000 ms and not inherited.
+    FLUENT_SETTING_OPTIONAL(TDuration, ReportInterval);
```

Comment:
```suggestion
    FLUENT_SETTING_OPTIONAL(std::chrono::milliseconds, ReportInterval);
```
Мы хотим совсем уйти от util в SDK. Так что в новом коде лучше сразу std::chrono использовать
И ниже по коду тоже